tracer/src/Datadog.Trace.Coverage.collector/CoverageCollector.cs

```diff
@@ -254,13 +231,6 @@ private void ProcessFolder(string folder, SearchOption searchOption)
             _logger?.Warning($"Processed {numAssemblies} assemblies in folder: {folder}");
         }
 
-        private void BackupFolder(string folder)
-        {
-            var destinationFolder = Path.Combine(folder, _dateTime.ToString("yyyyMMddHHmmss"));
-            _logger?.Debug($"Backup folder: {destinationFolder}");
-            Copy(folder, destinationFolder);
-        }
```

Comment:
Just wondering why you decided this wasn't required any more?

Reply:
We never use this backup, and we don't have plans to do a restore in CI. This was done more for debugging purposes.